src/Config.h

```diff
@@ -34,6 +34,8 @@ QT_BEGIN_NAMESPACE
 class QQmlEngine;
 QT_END_NAMESPACE
 
+class QSize;
```

Comment:
That should be in QT_{BEGIN,END}_NAMESPACE too I think

Reply:
OK

Reply:
Done, I force pushed it